packages/expo/src/provider/singleton/singleton.web.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * @deprecated Use `getClerkInstance` instead. `Clerk` will be removed in the next major version.
+ */
+export const clerk = globalThis?.window?.Clerk;
+export const getClerkInstance = globalThis?.window?.Clerk;
```

Comment:
Do we need to export those in order to have parity between web and native ? 
❌ Also `getClerkInstance` for native is a function.

Reply:
Yeah, we need them for parity so we don't have any differences between platforms.

Yeah you are right about `getClerkInstance` It's not used so that we can drop it

Reply:
Not sure which will be dropping, can you confirm ?

Reply:
We could drop it from the `singleton.web.tsx`, but I took a look again and it would be nicer to keep it just for having an understanding of how it works and what `getClerkInstance` does

Reply:
We should clear this up ☝️

Reply:
Hmm, okay just did some tests.

We will need to keep this one as we export both `getClerkInstance` and `clerk`, and cause we want to keep it isomorphic if we remove this and either of these is used it will result in an error